backend/src/test/java/com/woowacourse/gongcheck/core/domain/space/SpaceRepositoryTest.java

```diff
@@ -31,102 +34,163 @@ class SpaceRepositoryTest {
     @Autowired
     private SpaceRepository spaceRepository;
 
-    @Test
-    void Space_저장_시_생성시간이_저장된다() {
-        Host host = hostRepository.save(Host_생성("1234", 1234L));
 
-        LocalDateTime nowLocalDateTime = LocalDateTime.now();
-        Space space = spaceRepository.save(Space.builder()
-                .host(host)
-                .name(new Name("잠실"))
-                .build());
-        assertThat(space.getCreatedAt()).isAfter(nowLocalDateTime);
-    }
+    @Nested
+    class save_메서드는 {
+
+        private Host host;
+
+        @BeforeEach
+        void setUp() {
+            host = hostRepository.save(Host_생성("1234", 1234L));
+        }
 
-    @Test
-    void Host의_Space를_조회한다() {
-        Host host = hostRepository.save(Host_생성("1234", 1234L));
-        Space space1 = Space_생성(host, "잠실 캠퍼스");
-        Space space2 = Space_생성(host, "선릉 캠퍼스");
-        Space space3 = Space_생성(host, "양평같은방");
-        spaceRepository.saveAll(List.of(space1, space2, space3));
+        @Nested
+        class 입력받은_Space를_저장할_때 {
 
-        List<Space> result = spaceRepository.findAllByHost(host);
+            private Space space;
 
-        assertThat(result).hasSize(3);
+            @BeforeEach
+            void setUp() {
+                space = FixtureFactory.Space_생성(host, "잠실");
+            }
+
+            @Test
+            void 생성_시간도_함께_저장한다() {
+                LocalDateTime timeThatBeforeSave = LocalDateTime.now();
+                Space savedSpace = spaceRepository.save(space);
+                assertThat(savedSpace.getCreatedAt()).isAfter(timeThatBeforeSave);
+            }
+        }
     }
 
     @Nested
-    class Space를_조회할_때 {
+    class findAllByHost_메서드는 {
+
+        @Nested
+        class 입력받은_Host가_Space를_가지고_있을_때 {
+            private Host host;
+            private List<Space> expected;
+
+            @BeforeEach
+            void setUp() {
+                host = hostRepository.save(Host_생성("1234", 1234L));
+                Space space_1 = Space_생성(host, "잠실 캠퍼스");
+                Space space_2 = Space_생성(host, "선릉 캠퍼스");
+                Space space_3 = Space_생성(host, "양평같은방");
+                expected = spaceRepository.saveAll(List.of(space_1, space_2, space_3));
```

Comment:
```suggestion
                expected = spaceRepository.saveAll(List.of(
                    Space_생성(host, "잠실 캠퍼스"), Space_생성(host, "선릉 캠퍼스"), Space_생성(host, "양평같은방"));
```
취향차이입니다만 사용하지 않는 필드를 선언해두는 것은 불필요한 라인이라고 생각하는 편입니다 :)

Reply:
살짝 고민했던 부분인데, 반영하도록 하겠습니당~